imap_processing/ena_maps/utils/map_utils.py

```diff
@@ -99,6 +107,7 @@ def bin_values_at_indices(
     projection_grid_shape: tuple[int, int],
     projection_indices: NDArray,
     input_indices: NDArray | None = None,
+    spatial_axis: int = 0,
```

Comment:
I'd like to make this just be `-1` and not allow an option to change this if possible.

Reply:
Removed it as a parameter.